scope.go

```diff
@@ -273,6 +279,14 @@ func (s *standardScope) Reporter() StatsReporter {
 	return s.reporter
 }
 
+func (s *standardScope) Tags() map[string]string {
+	return s.tags
```

Comment:
Perhaps we should provide a copy of the tags map here so users of this library can't mangle the tags?  As it stands they could get the map then mutate it here which could cause all sorts of bad race conditions, etc.